ioctl/cmd/action/stake2change.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"encoding/hex"
+	"math/big"
+
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/action"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/validator"
+)
+
+// Multi-language support
+var (
+	stake2ChangeCmdUses = map[config.Language]string{
+		config.English: "change CANDIDATE_NAME BUCKET_INDEX [DATA]" +
+			"[-s SIGNER] [-n NONCE] [-l GAS_LIMIT] [-p GAS_PRICE] [-P PASSWORD] [-y]",
+		config.Chinese: "change 候选人姓名 桶索引 [数据]" +
+			" [-s 签署人] [-n NONCE] [-l GAS限制] [-p GAS价格] [-P 密码] [-y]",
+	}
+	stake2ChangeCmdShorts = map[config.Language]string{
+		config.English: "Change stake candidate",
+		config.Chinese: "改变质押候选人",
+	}
+)
+
+// stake2ChangeCmd represents the stake2 change command
+var stake2ChangeCmd = &cobra.Command{
+	Use:   config.TranslateInLang(stake2ChangeCmdUses, config.UILanguage),
+	Short: config.TranslateInLang(stake2ChangeCmdShorts, config.UILanguage),
+	Args:  cobra.RangeArgs(1, 3),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		err := stake2Change(args)
+		return output.PrintError(err)
+	},
+}
+
+func init() {
+	registerWriteCommand(stake2ChangeCmd)
+}
+
+func stake2Change(args []string) error {
+	var candidateName = args[0]
+	if err := validator.ValidateCandidateNameForStake2(candidateName); err != nil {
+		return output.NewError(output.ValidationError, "invalid candidate name", err)
+	}
+
+	bucketIndex, ok := new(big.Int).SetString(args[1], 10)
```

Comment:
index is uint64, it's more straightforward to use strconv.ParseInt()

Reply:
can you also fix for other cmds, if they have same issue?

Reply:
ok